### DISCUSSION TITLE
Couldnt find Purge CSS

### DISCUSSION THREAD

**Discussion by Jolaolu** [tailwindlabs/tailwindcss#2135] [Help] [answered]
I tried to rebuild and deploy my app to vercel and I got this

<img width="995" alt="Screenshot 2020-08-06 at 09 23 22" src="https://user-images.githubusercontent.com/37775521/89508979-97e3b300-d7c6-11ea-93dd-d501d759d6d5.png">
 

I've tried removing the `node_modules` and  reinstalling, I also tried changing the version of tailwind to the latest one but I got this 
```
error Couldn't find package "@fullhuman/postcss-purgecss@^2.1.2" required by "tailwindcss@^1.6.0" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by robbyrice** [comment, marked as answer]
The NPM servers were having some problems earlier today (or last night depending on the timezone) and that looks like the error you are seeing.
